requirements/local.in

```diff
@@ -0,0 +1,3 @@
+# Local development dependencies go here
```

Comment:
Could we use `dev.in` and `dev.txt` for these files?  That's what we're doing in other repos, and in `edx-platform` we use `local.in` for editable installations of packages kept inside the repository, which makes this naming a bit confusing.

Reply:
Yeah, makes sense. I will update.